### ISSUE TITLE
Feature Request : Adding Descriptions to Main Snapshot List

### ISSUE THREAD

**Issue by jtssi** [kopia/kopia#3510]
Hello everyone,

I would like to propose an improvement that, in my opinion, could be very useful for many users.

Currently, when I create backup snapshots of documents from other computers on the LAN with Kopia, I encounter a small difficulty: there is no direct way to know to whom each snapshot belongs. To address this issue, I am forced to use an external notepad to keep track of this information.

My suggestion is as follows: would it be possible to add an option to include a brief description under each snapshot location in the Kopia user interface?

I believe this feature would greatly improve the user experience by providing better organization and faster identification of snapshots, especially in an environment where multiple people are backing up their documents on the same network.

I am curious to know if other users feel the same need.

Exemple screen :
![2023-12-15_10 54 23](https://github.com/kopia/kopia/assets/71876429/03f6ee4d-f703-415d-b0a2-d3d475b97906)

**Comment by lupusA** [comment]
Hi @jtssi, 

the owner can be defined with a ```user@host``` scheme. Something like this:
![grafik](https://github.com/kopia/kopia/assets/37236531/837c6222-d289-4409-94ac-b70e8d2154fe)

Would that be enough to distinguish between users and hosts?

**Comment by jtssi** [comment]
No, in fact I run kopia from a server, and back up the machines via c$. So the user who backs up is the one on the server. We don't know which machine belongs to which IP.

**Comment by lupusA** [comment]
Hi @jtssi, 

i guess the ```cli``` already has that feature: https://kopia.io/docs/reference/command-line/common/snapshot-create/
You can add a description via ```--description``` to add a description to a snapshot. 

It seems that the UI cannot provide the same functionality yet. 

Cheers,

**Comment by lupusA** [comment]
> Hi @jtssi,
> 
> i guess the `cli` already has that feature: https://kopia.io/docs/reference/command-line/common/snapshot-create/
> 
> You can add a description to a snapshot via `--description`. It seems that the UI does not support it yet.
> 
> Cheers,

Nevermind, the description is for snapshots of that path. The requested feature is currently missing.

**Comment by calmwaves111** [comment]
> --description

I hope the UI version can support this feature as soon as possible. I believe it is a very useful feature, similar to Git's message
But the command line is too difficult for me
